### PR TITLE
Update TESTS.md

### PR DESCRIPTION
**Pull request by celiagomezdev**
Hi, I added some points to make it clearer to people who never made TDD or even coding before. Please let me know what do you think!

** edit
closes https://github.com/exercism/xswift/issues/202

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
+
+* Don't write code or modify the BobTest.swift file, this file is only to verify if the code you wrote in your Bob.swift file is right. Neither add in your working file anything from the testing file, just take it as a reference to figure out how to solve the exercise. _If you prefer, you can also work in an independent Playground file and then copy your final code into your working file (Bob.swift) and continue with the testing._
+
+* Then you just need to go to the Test Navigator (see image below) and click on the grey play button next to the main test (BobTest in this case). If the tests passed, they will show you a green checkmark, if not, a red x mark. Modify your code on your working file (Bob.swift) until all the tests succeed.
+
+<img src="https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/Art/twx-runtst-6_2x.png" width="250">
+
+* Some notes: XCTestCase is the XCTest subclass that contains our tests methods and XCTAssert lines are the expected results, with different types depending on what are you testing. Learn more about XCTAssert Methods and XCTesting itself here and here.
+
+* Now run the tests, and happy coding!
```
Comment (masters3d) [inline docs/TESTS.md]: here and here? :)

File: docs/TESTS.md [hunk 2]
```diff
@@ -1,19 +1,39 @@
...
+
+* Write the solution of the exercise in your working file (Bob.swift), in order to make the tests from the test file (BobTest.swift) succeed. Now the BobTest.swift file will show you some errors, but writing the right code in your working file Bob.swift should make them disappear.
+
+* Don't write code or modify the BobTest.swift file, this file is only to verify if the code you wrote in your Bob.swift file is right. Neither add in your working file anything from the testing file, just take it as a reference to figure out how to solve the exercise. _If you prefer, you can also work in an independent Playground file and then copy your final code into your working file (Bob.swift) and continue with the testing._
+
+* Then you just need to go to the Test Navigator (see image below) and click on the grey play button next to the main test (BobTest in this case). If the tests passed, they will show you a green checkmark, if not, a red x mark. Modify your code on your working file (Bob.swift) until all the tests succeed.
+
+<img src="https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/Art/twx-runtst-6_2x.png" width="250">
```
Comment (masters3d) [inline docs/TESTS.md]: host this image at githubusercontent because apple changes their links all the time

Comment (celiagomezdev) [inline docs/TESTS.md]: ok! how can I do this? I don't find any info about it! :)

File: docs/TESTS.md (hunk 1 above)
Comment (celiagomezdev) [inline docs/TESTS.md]: done! I added the links this way to be opened in a new tab, but I think it doesn't work anyway :/

File: docs/TESTS.md (hunk 2 above)
Comment (masters3d) [inline docs/TESTS.md]: I just cheat a little here. Upload the picture to an issue then just get the link and paste it in here. I wish there was a better way. 
https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

**Pull request merged**